content/docs/portals.md

```diff
@@ -27,34 +27,34 @@ render() {
 }
 ```
 
-However, sometimes it's useful to insert a child into a different location in the DOM:
+हालांकि, कभी कभार DOM में किसी अलग स्थान पर एक चाइल्ड को डालने की उपयोगिता होती है:
 
 ```js{6}
 render() {
-  // React does *not* create a new div. It renders the children into `domNode`.
-  // `domNode` is any valid DOM node, regardless of its location in the DOM.
+  // React एक नया div *नहीं* बनाता। वह चिल्ड्रन को `domNode` में रेंडर करता है।
+  // `domNode` कोई भी एक मान्य DOM नोड है, DOM में इसका स्थान मायने नहीं रखता।
   return ReactDOM.createPortal(
     this.props.children,
     domNode
   );
 }
 ```
 
-A typical use case for portals is when a parent component has an `overflow: hidden` or `z-index` style, but you need the child to visually "break out" of its container. For example, dialogs, hovercards, and tooltips.
+Portals का एक विशिष्ट उपयोग का मामला है जब एक पैरेंट कौम्पोनॅन्ट में `overflow: hidden` या `z-index` स्टाइल होता है, लेकिन आपको चाइल्ड को नेत्रहीन उसके कंटेनर से "बाहर" करने की आवश्यकता है। जैसे डॉयलोग्स, होवरकार्ड और टूलटिप्स।
```

Comment:
Portals का एक विशिष्ट उपयोग का मामला है -> Portals ke एक विशिष्ट उपयोग का उदाहरण and adjust the rest of the sentence to make it grammatically  correct

Reply:
जैसे -> जैसे ki